NDTensors/src/lib/GradedAxes/src/fusion.jl

```diff
@@ -83,24 +105,38 @@ end
 # Get the permutation for sorting, then group by common elements.
 # groupsortperm([2, 1, 2, 3]) == [[2], [1, 3], [4]]
 function blockmergesortperm(a::AbstractBlockedUnitRange)
-  # If it is dual, reverse the sorting so the sectors
-  # end up sorted in the same way whether or not the space
-  # is dual.
-  # TODO: Figure out how to deal with dual sectors.
-  # TODO: `rev=isdual(a)`  may not be correct for symmetries beyond `U(1)`.
-  ## return Block.(groupsortperm(nondual_sectors(a); rev=isdual(a)))
   return Block.(groupsortperm(blocklabels(a)))
```

Comment:
This is kind of a matter of taste, but I would prefer deleting the version defined for `UnitRangeDual` above and instead make this version more generic to handle both cases, for example I think we can just change this version to be defined as:
```julia
  return Block.(groupsortperm(blocklabels(nondual(a))))
```
For unit ranges that aren't dual to begin with, `nondual` will be a no-op.